html_css_dom/the_internet/README.md

```diff
@@ -0,0 +1,183 @@
+
+So far, all the code we have written has only existed and run on our local computers. Starting today, we're going to be learning about many things, but most importantly we'll talk about **requests** and how they work. They will be an essential part of our skillset as software engineers building web applications.
+
+## What is the internet?
+
+Short answer: it's a [series of tubes](https://www.youtube.com/watch?v=R8XSo0etBC4)
+
+Long answer:
+
+The internet is made up of many different types and layers of computers. These computers are connected by fiber optic cables that run around the world. Some even run [across the bottom of the ocean](https://en.wikipedia.org/wiki/Transatlantic_communications_cable).
+
+There is no one place that the internet lives - it's distributed across many different cities, though in many places it's especially concentrated. For example, in Ashburn, VA there are dozens of warehouses holding rows and rows of computers. Some estimates say that 70% of all US internet traffic flows through there, but there's no way to reliably measure that.
+
+Since the internet doesn't live only in one place, there are many challenges with routing internet traffic. There are special computers called DNS servers that just tell other computers where a website's traffic should be routed. Other computers (switches) do the actual routing - literally moving electrons from one fiber optic cable to another.
+
+Just like the internet doesn't exist in one place, there is no single company that runs the internet. There are only a handful of companies that manage networks though, and the networks they manage are called [Tier 1 networks](https://en.wikipedia.org/wiki/Tier_1_network). Sometimes they're called `backbone` networks, and they are extremely large and critical to the continued operation of the internet. If one of these backbones went down, everything connected to it would be unable to communicate - though the rest of the networks would be able to.
+
+![tier 1 network map](./assets/map-1.webp)
+
+![tier 1 network map 2](./assets/map-2.png)
+
+## Clients and servers
+
+You've probably heard the words `client` and `server` before. In a general sense, both of these words just refer to computers. A computer can be either a client, or a server, or sometimes (like your laptop) both!
+
+A `server` is a computer, listening for requests, and sending back responses. This response information can be many things, but is usually something like:
+
+- files: photos, js, css, html, or videos
+- data from a database
+
+A `client` is a computer that asks the server for information. A client can be one of many things - a laptop, a phone, a web browser, a program on your computer, a `node` script.
+
+You can run a `node` server on your computer and make requests to it from your same computer. We will do this a lot when we get into module 4. Where the server is and where the request comes from doesn't make any difference.
```

Comment:
Instead of saying `module 4` can we future proof this by saying something more vague? For example, `when we talk more about back end development`?